index.html

```diff
@@ -10,5 +10,14 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech-div">
+    <div class="tech-element-div">
+      <img class="img-tech" src="assets/react_icon.svg" alt="ReacjJS">
+      <span class="tech-descr">ReacjJS</span>
+      <span class="tech-descr"> <strong>Release: 2013</strong></span>
+      <span class="tech-descr"> React is a declarative, component-based Javascript
```

Comment:
Try to mantain things in one line, if text is too long it would be like 
`<span> (newline)
    Lot of text (newline)
</span`

Reply:
will be corrected on the next commit